pkg/minikube/driver/install.go

```diff
@@ -48,10 +48,11 @@ func InstallOrUpdate(name string, directory string, v semver.Version, interactiv
 	spec.Timeout = 10 * time.Minute
 	glog.Infof("acquiring lock: %+v", spec)
 	releaser, err := mutex.Acquire(spec)
+	start := time.Now()
+	defer lock.Release(releaser, "driverInstall", start)
```

Comment:
I don't think this is correct. One should never attempt to release locks if they fail to be set.